### PR TITLE
in dagstermill test, don't dump entire dagster event to json

### PR DESCRIPTION
**Pull request by sryza**
## Summary & Motivation

In general, DagsterEvents are not always json-dumpable (e.g. they contain enums), but this test assumes they are. This causes problems when moving from `NamedTuple` (which are json-dumpable) to `DagsterModel` (which are not json-dumpable) for classes within `DagsterEvent`s.

### CODE REVIEW

File: python_modules/libraries/dagstermill/dagstermill_tests/test_logging.py [hunk 1]
```diff
@@ -30,7 +30,16 @@ def __init__(self, file_path):
...
     def emit(self, record):
         with open(self.file_path, "a", encoding="utf8") as fd:
-            fd.write(seven.json.dumps(record.__dict__) + "\n")
+            fd.write(
+                seven.json.dumps(
+                    {
+                        "dagster_meta": {
+                            "orig_message": record.__dict__["dagster_meta"]["orig_message"]
+                        }
+                    }
```
Comment (alangenfeld) [inline python_modules/libraries/dagstermill/dagstermill_tests/test_logging.py:39]: how did you end up with this? leave a comment at least

**Review by alangenfeld — APPROVED** [review]
im not even sure what exactly is under test with this complex set-up

**Comment by sryza** [comment]
> im not even sure what exactly is under test with this complex set-up

I think it's that stuff logged from within the notebook uses the logger configured on the job

File: python_modules/libraries/dagstermill/dagstermill_tests/test_logging.py (hunk 1 above)
Comment (sryza) [reply python_modules/libraries/dagstermill/dagstermill_tests/test_logging.py]: was trying to fit the asserts below, but we don't actually need this nested structure. I changed to simplify.

**Pull request merged**